utils-macro/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "miden-node-utils-macro"
```

Comment:
I think we are using this crate for testing only. Would a more proper name would be `miden-node-test-utils`?